config/routes.rb

```diff
@@ -6,8 +6,7 @@
     resources :sessions
     resources :categories
     resources :games
-    get 'user_games', to: 'games#user_games'
-    get 'user_game/:id', to: 'games#user_game'
+    #get 'user_game/:id', to: 'games#user_game'
```

Comment:
Are you holding onto this route on purpose or did you just forget to delete it after testing?

Reply:
just forgot haha, I'll remove that extra line in the next PR!